ext/RMagick/rmimage.c

```diff
@@ -12393,10 +12393,8 @@ Image_sharpen_channel(int argc, VALUE *argv, VALUE self)
             raise_ChannelType_error(argv[argc-1]);
     }
 
-    new_image = rm_clone_image(image);
-
     exception = AcquireExceptionInfo();
-    (void) SharpenImageChannel(new_image, channels, radius, sigma, exception);
+    new_image = SharpenImageChannel(image, channels, radius, sigma, exception);
```

Comment:
@Watson1978 does `SharpenImageChannel` mutate the image passed in? If not, then how was this method working to begin with?

Reply:
I checked `SharpenImageChannel()` and it's sub-function and looks for me that they does not change the `image` arugment's parameter.

Reply:
Okay, let me see if I have this right. Before your change this is what looks to me like is happening:

1) we clone the image as `new_image`
2) we pass the clone into `SharpenImageChannel`, which *does not* modify `new_image`, but returns a new copy which we ignore
3) we return another new image based on `new_image`, which is exactly the same as the `image` that was passed in.

Does this sound correct? I tested this locally and was able to confirm that it seems to work this way. I don't think this is a breaking change. I think you fixed an existing bug where `sharpen_channel` was not doing anything.

Before this PR (image does not change):

![before_fix](https://user-images.githubusercontent.com/80543/53066680-d1ffae80-3485-11e9-9ba0-c35073f79e7c.png)

After this PR (image changes):

![after_fix](https://user-images.githubusercontent.com/80543/53066689-d9bf5300-3485-11e9-8cdf-309492580b46.png)

Reply:
Thank you for your check and you're right.

I just used `breaking change` for changing behavior, so we don't need to describe to ChangeLog.

Reply:
Great! Glad I understand.